modules/veteran_confirmation/app/controllers/veteran_confirmation/v0/veteran_status_controller.rb

```diff
@@ -5,8 +5,70 @@
 module VeteranConfirmation
   module V0
     class VeteranStatusController < ApplicationController
+      before_action :validate_body
+
       def index
-        render json: { hit_it: 'yep' }
+        body = JSON.parse(string_body)
+
+        attributes = {
+          ssn: body['ssn'],
+          first_name: body['first_name'],
+          last_name: body['last_name'],
+          birth_date: Date.iso8601(body['birth_date']).strftime('%Y%m%d')
+        }
+
+        status = StatusService.new.get_by_attributes(attributes)
+
+        render json: { veteran_status: status }
+      end
+
+      private
+
+      def validate_body
+        raise error_klass('Body must not be empty') if string_body.blank?
+
+        body = JSON.parse(string_body)
```

Comment:
We should probably rescue JSON parse errors and tell the user something meaningful. Just in case

Reply:
I made changes to how we're parsing the JSON body. It is automatically parsed into params, so we don't need to handle it manually.